flocker/node/agents/ebs.py

```diff
@@ -160,7 +282,7 @@ def create_volume(self, dataset_id, size):
         Open issues: https://clusterhq.atlassian.net/browse/FLOC-1792
         """
         requested_volume = self.connection.create_volume(
-            size=int(Byte(size).to_GB().value), zone=self.zone)
+            size=int(Byte(size).to_GiB().value), zone=self.zone)
```

Comment:
Added link to FLOC-1874 for EBS and Cinder. Added a note to FLOC-1874 as well. Thanks!